listener/subgraph_definition/schema.graphql

```diff
@@ -1,3 +1,8 @@
 type Deposit @entity {
     id: ID!
+    accountId: Int!
+    tokenId: Int!
+    amount: BigInt!
```

Comment:
Safe to assume the graph treats this as a string?

Reply:
We don't really care how they treat it on byte level as on a code level we work with the BigDecimal abstraction (but yes, turns out they store it as a string similar to how we do it atm in mongo).